triage/models/transformers.py

```diff
@@ -0,0 +1,67 @@
+# coding: utf-8
+
+import warnings
+
+import numpy as np
+
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.utils import check_array
+from sklearn.utils.validation import FLOAT_DTYPES
+
+DEPRECATION_MSG_1D = (
+    "Passing 1d arrays as data is deprecated in 0.17 and will "
+    "raise ValueError in 0.19. Reshape your data either using "
+    "X.reshape(-1, 1) if your data has a single feature or "
+    "X.reshape(1, -1) if it contains a single sample."
+)
+
+class DsappCutOff(BaseEstimator, TransformerMixin):
```

Comment:
This, too, could use a better name. Just calling it 'Cutoff' might work, though the terms 'Crop' and 'Clip' also kind of describe what is being done.